### ISSUE TITLE
feature request: how to properly work with arrow

### ISSUE THREAD

**Issue by JosephBARBIERDARNAL** [holtzy/The-Python-Graph-Gallery#330]
code snippet

```
import matplotlib.pyplot as plt
import matplotlib.patches as patches
def add_arrow(head_pos, tail_pos, ax):
    style = "Simple, tail_width=0.01, head_width=1, head_length=2"
    kw = dict(arrowstyle=style, color="k", linewidth=0.2)
    arrow = patches.FancyArrowPatch(
        tail_pos, head_pos,
        connectionstyle="arc3,rad=.5",
        **kw
    )
    ax.add_patch(arrow)


# test
fig, ax = plt.subplots(figsize=(5, 5))
ax.set_xlim(0, 10)
ax.set_ylim(0, 10)
add_arrow((1, 1), (5, 5), ax)
plt.show()
```

**Comment by holtzy** [comment]
All good.

- Add in the matplotlib section
- Add 3 links somewhere in the gallery.

**Comment by JosephBARBIERDARNAL** [comment]
fixed with #339
